dashing.gemspec

```diff
@@ -1,7 +1,7 @@
 # -*- encoding: utf-8 -*-
 
 Gem::Specification.new do |s|
-  s.name        = 'dashing'
+  s.name        = 'dashing-jruby'
```

Comment:
Agreed, the project should remain available as the gem `dashing`.

However, the remainder of this PR has merit and could be considered. I have no allegiances to Thin.